service_management/azure/test/integration/cloud_service/cloud_delete_test.rb

```diff
@@ -15,30 +15,29 @@
 require 'integration/test_helper'
 
 describe Azure::CloudServiceManagementService do
-  include Azure::Core::Utility
-
   subject { Azure::CloudServiceManagementService.new }
-
+  let(:cloud_service_name) { 'testcloudservice2375' }
   let(:options) do
     {
-      location: 'West US',
-      description: 'Test'
+      location: 'East US',
+      description: 'Cloud service test description'
     }
   end
 
   before do
     Azure::Loggerx.expects(:puts).returns(nil).at_least(0)
+    VCR.insert_cassette "cloud_service/#{name}"
+    subject.create_cloud_service(cloud_service_name, options)
   end
 
-  describe '#delete_cloud_service' do
-    before do
-      @cloud_name = random_string('test-service-cloud', 10)
-      subject.create_cloud_service(@cloud_name, options)
-    end
+  after do
+    VCR.eject_cassette
+  end
 
-    it 'Deletes the cloud service in Microsoft Azure.' do
-      subject.delete_cloud_service(@cloud_name)
-      cloud_service = subject.get_cloud_service(@cloud_name)
+  describe 'cloud service' do
+    it 'should delete the cloud service' do
```

Comment:
Should have one file containing create and delete cloud service tests? We end up doing almost the same in each file.